frontend/src/libs/packages/notification/libs/options/notification.options.default.ts

```diff
@@ -0,0 +1,16 @@
+import { type ToastOptions, toast } from 'react-toastify';
+
+import { NotificationTheme } from '../enums/notification.theme.enum.js';
+
+const defaultNotificationOptions: ToastOptions = {
```

Comment:
also create separate type for toast options, describe inside only those which we are planning to use. Now a lot of default options I suppose we will not use